src/Tasks/Microsoft.NET.Build.Tasks/Resources/Strings.resx

```diff
@@ -249,4 +249,10 @@
   <data name="RuntimeIdentifierWasNotSpecified" xml:space="preserve">
     <value>Specify a RuntimeIdentifier</value>
   </data>
+  <data name="FileNameIsTooLong" xml:space="preserve">
+    <value>Given File Name '{0}' is longer than 1024 bytes</value>
+  </data>
+  <data name="HostIsNotNuetered" xml:space="preserve">
+    <value>The App host does not have the embedded hash value of string '{0}'</value>
```

Comment:
As of now, this will report 'foobar' to user which would be a world ready tenet blocker and really embarrassing if we ever have to explain this error.

What is the point of using a hash for this marker. Can't we just generate a guid?

I think the user facing message for this should not go in to the implementation details of the replacement. This is basically an invalid (corrupted?) apphost, right?